stable/influxdb/templates/statefulset.yaml

```diff
@@ -58,9 +62,19 @@ spec:
         - name: opentsdb
           containerPort: {{ .Values.config.opentsdb.bind_address }}
         {{- end }}
+        {{- if .Values.enterprise.enabled }}
+        - name: rpc
+          containerPort: {{ .Values.config.rpc.bind_address }}
+        - name: meta
+          containerPort: {{ .Values.config.meta.bind_address }}
+        {{- end }}
         {{- if .Values.env }}
         env:
 {{ toYaml .Values.env | indent 10 }}
+        {{- if .Values.enterprise.enabled }}
+          - name: INFLUXDB_HOSTNAME # Values.env's HOSTNAME isn't fundamentally different from $HOSTNAME, but this way weg get a distinguished name at runtime.
```

Comment:
Not familiar with enterprise so not sure how/what this is used for, but would "{{ template "influxdb.fullname" . }}-<stateful set replica number> (Which is value for $HOSTNAME) not be sufficient?

Reply:
Sorry, "distinguished" wasn't a great fit here. I meant like a FQDN. InfluxDB Enterprise needs names to be resolvable.